nextflow_schema.json

```diff
@@ -124,6 +124,11 @@
                     "description": "The minimum percent identity when identifying repeats.",
                     "minimum": 0,
                     "maximum": 100
+                },
+                "skip_density_filter": {
+                    "type": "boolean",
```

Comment:
Can you add a `"default": false` here to match default in nextflow.config

Reply:
As discussed, `nf-core schema build` doesn't like `"default": false` here.

Reply:
Thanks. Yes. Makes sense to leave default out then.